### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by owen-adams2021**
image

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey dude! Please add a screenshot of your output in mobile view

**Comment by owen-adams2021** [comment]
image

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+* {
+  font-family: 'Lexend Mega', sans-serif;
+  background-color: #f5f5f5;
```
Comment (codingcodymiller) [inline css/styles.css:3]: You shouldn't apply a background color to every element like this. You should add background colors to just the relevant section.

Comment (codingcodymiller) [reply css/styles.css:3]: You could try `body`

File: index.html [hunk 2]
```diff
@@ -1,25 +1,58 @@
...
+  <main class="container">
+    <h1 class="column-full">New Entry</h1>
+    <div data-entry="view-form">
+      <form class="code-journal" id="journal-input">
+        <div class="row">
+          <div class="column-half" id="column-share">
+            <img src="images/placeholder-image-square.jpg" alt="journal-image" id="image">
+          </div>
+          <div class="column-half">
+            <label for="image-url"><p>Image URL</p>
```
Comment (codingcodymiller) [inline index.html:32]: You don't need this `p` element, since you already made `label` elements block.

File: index.html [hunk 3]
```diff
@@ -1,25 +1,58 @@
...
+              <input name="photoUrl" type="url" class="column-full" id="image-url" required>
+            </label>
+            <label for="entry-title"><p>Title</p>
```
Comment (codingcodymiller) [inline index.html:35]: You don't need this `p` element, since you already made `label` elements block.

File: index.html [hunk 4]
```diff
@@ -1,25 +1,58 @@
...
+              <input name="journalTitle" type="text" class="column-full" id="entry-title" required>
+            </label>
+          </div>
+        </div>
+        <div class="column-full">
+          <label class="notes" for="entry-content"><p>Notes</p>
```
Comment (codingcodymiller) [inline index.html:41]: You don't need this `p` element, since you already made `label` elements block.

File: index.html [hunk 5]
```diff
@@ -1,25 +1,58 @@
...
+            <textarea name="notes" type="text" class="column-full" id="entry-content" required>
+            </textarea>
+          </label>
+        </div>
+        <div class="column-full">
+          <label for="submit-button">
```
Comment (codingcodymiller) [inline index.html:47]: Submit buttons don't need a label. Their label is the word "Submit"

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far! Please see my notes

**Comment by owen-adams2021** [comment]
Updated screenshots of the journal entry form

Mobile:

image

Desktop:

image

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳

**Pull request merged**